Model/Push.php

```diff
@@ -1023,8 +1023,11 @@ public function processSucceededPush($newStatus, $message)
                 || $paymentMethod->getConfigData('order_email', $store)
             )
         ) {
-            $this->logging->addDebug(__METHOD__ . '|sendemail|');
-            $this->orderSender->send($this->order);
+            $this->logging->addDebug(__METHOD__ . '|sendemail|' . var_export(
+                $this->configAccount->getOrderConfirmationEmailSync($store),
+                true)
+            );
+            $this->orderSender->send($this->order, $this->configAccount->getOrderConfirmationEmailSync($store));
```

Comment:
try/catch for email sending would be useful.

Reply:
yeah, but to be honest I don't write new functionality for sending, it's just a new parameter adding
I think you can create new task for refactoring to wrap all mailings in try/catch because there are many usages